pyEpiabm/pyEpiabm/sweep/queue_sweep.py

```diff
@@ -23,6 +23,5 @@ def __call__(self, time):
                 # Get takes person from the queue and removes them, so clears
                 # the queue for the next timestep.
                 # Update the infection status
-                person.update_status(InfectionStatus.Exposed)
-                person.next_infection_status = InfectionStatus.InfectMild
+                person.next_infection_status = InfectionStatus.Exposed
```

Comment:
This change doesn't seem to match the docstring, is the queue sweep processing people becoming exposed or infectious?